draft-ietf-ohai-ohttp.md

```diff
@@ -1240,17 +1249,17 @@ proxies close to servers was most effective in minimizing additional latency.
 ## Resource Mappings {#proxy-state}
 
 This protocol assumes a fixed, one-to-one mapping between the Oblivious Proxy
-Resource and the Oblivious Request Resource. This means that any encapsulated
+Resource and the Oblivious Encapsulation Resource. This means that any encapsulated
```

Comment:
tangent(?): why are these instances capitalized and others aren't?

Reply:
That's a bug that we should fix separately 👍